sebex/language/elixir/__init__.py

```diff
@@ -103,7 +103,10 @@ def publish(self, project: ProjectHandle) -> bool:
             return False
 
         with operation('Publishing for real'):
-            proc = popen(['mix', 'hex.publish', '--yes'], log_stdout=True, cwd=project.location)
+            if 'sebex_test' in str(project):
```

Comment:
again; hardcoded behaviour which should be rather controlled via config

Reply:
Also, I guess the only reason you pass `--replace` here is that you are doing integration testing on real private Hex.pm organization? Did you consider running some local Hex.pm instance in Docker or smth for testing purposes in CI?

Reply:
Yes I tried hosting a local Hex instance in a Docker container and while it's possible to `hex.fetch <package> --repo=<local_repo>` without a problem the `hex.publish` command doesn't work this way - it's necessary to build the elixir project, then manually copy the tarball over to the local Hex' `public/tarballs` folder while dealing with version collisions and then rebuild the registry of the local hex and rehost it. Do you think it's worth it to simulate the unimplemented `hex.publish` this way?

Reply:
that's a pity that it doesn't work out of the box 😢 

yeah, if so then I wouldn't bother now with this. I think I would do a global config flag for this in manifest then, smth like:

```yaml
repositories:
...
config:
  hex:
    allow_replace_on_publish: yes
```